components/match2/commons/match_summary_base.lua

```diff
@@ -204,7 +221,7 @@ end
 ---@field root Html
 ---@field headerElement Html?
 ---@field bodyElement Widget|Html?
----@field commentElement Widget?
+--@field commentElement Widget?
```

Comment:
why?

Reply:
If i only knew... looks like i misclicked/typed in vscode. Need to align my hotkeys between machines